README.md

```diff
@@ -89,6 +102,10 @@ To use custom uniforms define those directly in your shader, giving an initial v
 #iUniform color3 my_color = color3(1.0) // This will be editable as a color picker
 #iUniform vec2 position_in_2d = vec2(1.0) // This will expose two text fields
 #iUniform vec4 other_color = vec4(1.0) in { 0.0, 1.0 } // This will expose four sliders
+
+ `iViewMatrix`, a mat4 with the view matrix. Must use directive `#iFirstPersonControls` in order to control, otherwise it remains identity.
+ Controls:
+ <b>WASD</b> move, <b>R|F</b> up | down, <b>Q|E</b> roll, <b>up|down</b> pitch, <b>left|right</b> yaw
```

Comment:
These controls are a bit unusual for me personally, but I know everyone will have different preferences. So I was wondering whether we can add keybindings for these. From what I can see keybindings are only properly supported for actual commands so it might not be possible.